dev-client/src/screens/SoilScreen/utils/renderValues.ts

```diff
@@ -29,26 +30,39 @@ export const renderDepthInterval = ({
   return `${label ? `${label}: ` : ''}${start}-${end} cm`;
 };
 
+const colorMethods = [
+  'colorChroma',
+  'colorHue',
+  'colorHueSubstep',
+  'colorValue',
+] as const;
+
 // TODO: finish this method for other inputs
 export const pitMethodSummary = (
   t: TFunction,
   soilData: DepthDependentSoilData | undefined,
   method: SoilPitMethod | 'rockFragmentVolume',
 ): {complete: boolean; summary?: string} => {
-  if (method === 'soilTexture') {
-    return {
-      complete: typeof soilData?.texture === 'string',
-      summary: soilData?.texture
-        ? t(`soil.texture.class.${soilData?.texture}`)
-        : undefined,
-    };
-  } else if (method === 'rockFragmentVolume') {
-    return {
-      complete: typeof soilData?.rockFragmentVolume === 'string',
-      summary: soilData?.rockFragmentVolume
-        ? t(`soil.texture.rockFragment.${soilData.rockFragmentVolume}`)
-        : undefined,
-    };
+  if (soilData === undefined) {
+    return {complete: false};
+  }
+  let summary: string | undefined;
+  if (method === 'soilTexture' && soilData.texture) {
+    summary = t(`soil.texture.class.${soilData?.texture}`);
+  } else if (method === 'rockFragmentVolume' && soilData.rockFragmentVolume) {
+    summary = t(`soil.texture.rockFragment.${soilData.rockFragmentVolume}`);
+  } else if (
+    method === 'soilColor' &&
+    colorMethods.every(m => typeof soilData[m] === 'string')
+  ) {
+    summary = t(
+      'soil.color.rendered',
+      fromEntries(
+        colorMethods.map(m => [m, t(`soil.color.${m}.${soilData[m]}`)]),
+      ),
+    );
   }
-  return {complete: false};
+  return {complete: summary !== undefined, summary};
 };
+
+export const munsell2RGB = (_: DepthDependentSoilData) => '#b48677';
```

Comment:
question: where does `#b48677` come from? should it be in a config / constant / theme file?